ui/src/components/Absentee/Absentee.vue

```diff
@@ -12,20 +12,18 @@
           >
           </cv-checkbox>
 
-          <h2 class="aside__header">{{ $t('absenteeTitle') }}</h2>
+          <h2 class="aside__header" style="margin-top: 2rem">{{ $t('absenteeTitle') }}</h2>
           <p class="aside__paragraph">
             {{ $t('absenteeDesc') }}
           </p>
 
           <territory-info v-if="info.mail_in.territory" />
 
           <!-- Request deadline -->
-          <div class="register-info" v-if="info.mail_in.request_deadline">
-            <span
-              >{{ $t('registrationDeadline') }}
-              <span class="days-left">{{ daysLeft(info.mail_in.request_deadline) }}</span>
-              . ( {{ niceDate(info.mail_in.request_deadline) }}) {{ $t('actNow') }}
-            </span>
+          <div class="register-info-deadline" v-if="info.mail_in.request_deadline">
+            {{ $t('registrationDeadline') }}
```

Comment:
consider a span with class as well to allow detailed targeting, as should the span with no class have a class?   
but not important at the moment

Reply:
Nice catch. I'm going to redo those since the translation is broken by that pattern anyway. You are not supposed to string together translatable bits like A + B + C to form a sentence like that anyway. On the page I working on now I use a markdown string like this: `"Early voting begins **{medDate}**, *{relativeDays}*."` The right bits get substituted in in the right context and the markdown parts get rendered as `<em>` and `<strong>` tags that I can target with the styles.